### PR TITLE
feat(cb2-12679): update activty type and add enums

### PR DESCRIPTION
**Pull request by LGin-BJSS**
## Ticket title

_One line description_

[CB2-XXXX](https://dvsa.atlassian.net/browse/CB2-XXXX)

## Changelog

-

### CODE REVIEW

File: json-schemas/v1/activity/index.json [hunk 1]
```diff
@@ -32,9 +62,6 @@
 		"endTime": {
 			"type": "string"
 		},
-		"parentId": {
-			"type": "string"
-		},
 		"waitReason": {
```
Comment (cb-cs) [inline json-schemas/v1/activity/index.json:68]: Did we move away from creating an enum for waitReasons? I know it was mentioned in one of the chats.

Comment (LGin-BJSS) [reply json-schemas/v1/activity/index.json:68]: Got dragged in to looking at payments service stuff. will add that now.

Comment (cb-cs) [reply json-schemas/v1/activity/index.json:68]: Okay great thanks.

**Review by cb-cs — APPROVED** [review]
I know this is going to be pulled into CB2-12677 so there is only one major version bump, I think we can validate this is working as expected once activities pulls it in and uses it, but looks right to me.

**Pull request closed without merging**